app/views/editLiability/dataOfDisposal.scala.html

```diff
@@ -17,10 +17,8 @@
 @import models._
 @import config.ApplicationConfig
 @(disposeLiabilityForm: Form[DisposeLiability], oldFormBundleNo: String, backLink: Option[String], periodKey: Int)(implicit authContext: StandardAuthRetrievals, messages: Messages, request: Request[AnyContent], appConfig: ApplicationConfig)
-@import uk.gov.hmrc.play.views.html._
 @import uk.gov.hmrc.play.views.html.helpers._
 @import views.html.helpers._
-@import views.html.helpers.error._
```

Comment:
How do we know that these entries can be removed - my version of IntelliJ does not specify which imports are or are not used for views ?